### ISSUE TITLE
Another Slimefun conflict [Necrotic Skull]

### ISSUE THREAD

**Issue by jordanwilliams1** [meiskam/PlayerHeads#17]
Slimefun Necrotic Skull allows you to duplicate wither skulls. Just spawn one in with slimefun and break it with this plugin loaded. I've tried disabling the config options but the bug still presents0. 

https://i.gyazo.com/7456c8c9e664e8e72f260e474dc44005.gif

antideathchest: false
dropboringplayerheads: false
fixdroppedheads: false
pkonly: false

**Comment by crashdemons** [comment]
After a lot of testing I've determined that the reason this occurs is because of a feature added to the plugin a long time ago to remove a conflict with NoCheatPlus.

This feature exempts a user from NCP rules, **sends a test fake blockbreak event**, then unexempts a user - this is in order to detect if the user actually can break blocks in the area outside of NCP rules (or other drop protection).

I've made a commit ( df216dd39d0f7de2ec868fafa1fda824a93ce535 ) and a development build https://ci.meme.tips/job/PlayerHeads-5.2/16/

try using this build and the following command:
_/ph config set pretestblockbreak false_

Changing this setting will break support for NoCheatPlus though.

As an alternative solution, I've also tried to look on the block for tags indicating that the head belonged to slimefun but.... it doesn't seem there are any to check https://i.imgur.com/nhCS3MX.png
